com.redhat.patchpal.gui.yaml

```diff
@@ -0,0 +1,38 @@
+app-id: com.redhat.patchpal.gui
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+  - org.freedesktop.Sdk.Extension.llvm20
+command: patchpal
+separate-locales: false
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --filesystem=xdg-config
```

Comment:
why does it need access to `xdg-config` ?

Reply:
> For the git config, see the README.

it should be xdg-config/git then.